tsconfig.server.json

```diff
@@ -0,0 +1,18 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
+{
+  "extends": "./tsconfig.app.json",
+  "compilerOptions": {
+    "outDir": "./out-tsc/server",
+    "target": "es2016",
+    "types": [
+      "node"
+    ]
```

Comment:
Are these not imported from the `tsconfig.app.json`?

Reply:
Part of the SSR template